TitanRobotics2023/src/main/java/frc/robot/actors/DriveControl.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.actors;
+
+//import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import frc.robot.data.PortMap;
+import edu.wpi.first.wpilibj.motorcontrol.MotorController;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import edu.wpi.first.wpilibj.motorcontrol.PWMVictorSPX;
+
+//This class handles all drive control actions
+
+public class DriveControl
+{
+    private final MotorController motor_frontLeft;
+    private final MotorController motor_rearLeft;
+    private final MotorController motor_frontRight;
+    private final MotorController motor_rearRight;
+    private final MotorControllerGroup motorGroup_left;
```

Comment:
Remove this and the group below it since it's never used along with commented out line 19